README.md

```diff
@@ -1,51 +1,230 @@
+
+**2. Clone this project on your computer:**
+
+- Go to your profile. You will find forked repo named **_open-source-practice_**. go to the repo by clicking on it.
+- Click on the green Code button, then either the HTTPS or SSH option, and, click the icon to copy the URL. Now you have a copy of the project. Thus, you can play around with it locally on your computer.
+
+- Run the following commands into a terminal window (Command Prompt, Powershell, Terminal, Bash, ZSH). Do this to download the forked copy of this repository to your computer. ![git-clone](https://github.com/AmanxUpadhyay/hacktoberfest-practice/assets/76415079/4d600e25-83b1-4e8f-9325-f1adc4f8ce3d)
+
+```bash
+  git clone https://github.com/YOUR_GITHUB_USERNAME/hacktoberfest-practice.git
+```
 
-* Add folder.
-* Commit and push your code.
+- Switch to the cloned folder. You can paste this command into the same terminal window. ![opening-vscode](https://github.com/AmanxUpadhyay/hacktoberfest-practice/assets/76415079/1a8b350d-0c96-461a-85f4-a59185aed6b6)
 
-```markdown
-git add .
-git commit -m 'Commit message'
-git push origin branch-name
+```bash
+  cd hacktoberfest-practice
 ```
 
-* Create a new pull request from your forked repository (Click the `New Pull Request` button located at the top of your repo)
-* Wait for your PR review and merge approval...
-* __Star this repository__ if you had fun contributing!
+**3. Create a new branch:**
+
+- Your username would make a good branch because it's unique. ![git-checkout](https://github.com/AmanxUpadhyay/hacktoberfest-practice/assets/76415079/68ab0380-c731-4e67-bccc-b666da5dd174)
+
+```bash
+  git checkout -b <name-of-new-branch>
+```
+
+**4. Edit:**
+
+- Open the `README.md` file
+
+- **Add your name to the section that matches your Initial in [this list](https://github.com/EddieHubCommunity/open-source-practice#hacktoberfest-community), make sure that your name is in alphabetical order. Then save your changes.**
+
+- For example
+  `- [Full Name](https://github.com/your-username)`
+
+**5. Stage your changes:**
+
+```bash
+  git add README.md
+```
+
+or
+
+```bash
+  git add .
+```
+
+**6. Commit the changes:**
+![git-commit](https://github.com/AmanxUpadhyay/hacktoberfest-practice/assets/76415079/9a6a58f3-ed0b-4bc2-b749-48baede77835)
+
+```bash
+  git commit -m "Add <your-github-username>"
+```
+
+- Check the status of your repository.
+
+```bash
+  git status
+```
+
+- The response should be like this:
+  ![git-status](https://github.com/AmanxUpadhyay/hacktoberfest-practice/assets/76415079/d3692bcd-02d9-47d7-9e6c-b386b0a6c72d)
+
+```bash
+On branch <name-of-your-branch>
+nothing to commit, working tree clean
+```
+
+**7. Pushing your repository to GitHub:**
+
+```bash
+  git push origin <name-of-your-branch>
+```
+
+or
```

Comment:
remove the lines from 196 to  200, they shouldn't push code to the main branch so no need to mention this.

Reply:
done